client/lomas_client/client.py

```diff
@@ -202,7 +221,7 @@ def get_previous_queries(self) -> Optional[List[dict]]:
                         model = base64.b64decode(query["response"]["result"]["model"])
                         query["response"]["result"]["model"] = pickle.loads(model)
                     case _:
```

Comment:
maybe name the case something like `unknowType` and re-use it in the Log/Error ?